python-checks/src/main/java/org/sonar/python/checks/hotspots/ClearTextProtocolsCheck.java

```diff
@@ -134,9 +134,9 @@ public static Stream<String> getClassFQNFromArgument(List<Argument> arguments) {
       .map(TreeUtils.toInstanceOfMapper(RegularArgument.class))
```

Comment:
Small detail as it is not part of this PR but as you mentioned the access modifier is incorrect on this method. Maybe could quickly correct it.